client/platform/web-girder/main.ts

```diff
@@ -42,10 +43,13 @@ Promise.all([
     router,
     store,
     vuetify,
-    provide: { girderRest, notificationBus, vuetify },
+    provide: { girderRest, vuetify },
     render: (h) => h(App),
   })
     .$mount('#app')
     .$snackbarAttach()
     .$promptAttach();
+
+  /** Start notification stream if everything else succeeds */
+  registerNotifications(girderRest).connect();
```

Comment:
Should we rely on the `login` signal from the `restClient` to connect instead of doing it here?
If you sit at the login screen it is constantly attempting to reconnect right now because there is a 401 error so it's in the error/reconnect loop.  I could be missing something though.